PC/VS9.0/pyproject.vsprops

```diff
@@ -12,7 +12,7 @@
 		InlineFunctionExpansion="1"
 		EnableIntrinsicFunctions="true"
 		AdditionalIncludeDirectories="..\..\Include; ..\..\PC"
-		PreprocessorDefinitions="_WIN32"
+		PreprocessorDefinitions="_WIN32;WIN32"
```

Comment:
I'm not sure that this change is still needed with expat 2.2.1. Would you mind to check please?